### PR TITLE
Fixes #3419 - Make image upload keyboard navigable

### LINKED ISSUE #3419: [A11Y] Unable to upload screenshot using keyboard buttons
**Browser / Version**: Firefox Nightly 81.0a1 (2020-07-27) 
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Using "Tab" key, navigate to the URL field and fill it in. 
3. Using "Tab" key, navigate to "Confirm" button.
4. Reach the "Upload image" section and try to upload an image using keyboard keys.
5. Observe behavior.

**Expected Behavior:**
Uploading an image can be performed.

**Actual Behavior:**
Uploading an image can't be performed.

**Notes:**
1. Screenshot attached.
image
2. According to "Inspector>Accessibility" tab, the  "Interactive elements must be focusable" (Read more here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard).
image

**Watchers:**
@softvision-oana-arbuzov
@cipriansv

### PR DESCRIPTION
**Pull request by miketaylr**
r? @ksy36

### CODE REVIEW

**Comment by miketaylr** [comment]
`Error: socket hang up` and `Error: connect ECONNREFUSED 127.0.0.1:4444`. Hooboy.

**Comment by miketaylr** [comment]
I might have crashed Marionette somehow:

```
× firefox on mac 18.7.0 - Image Uploads (non-auth) - double image select works (1.994s)
    JavaScriptError: [POST http://localhost:4444/wd/hub/session/533725cd-78a4-a842-acac-7e848c823af5/execute / {"script":"postMessage(\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABkAAAAZCAIAAABLixI0AAAACXBIWXMAAA7DAAAOwwHHb6hkAAAAB3RJTUUH3gYSAig452t/EQAAAClJREFUOMvtzkENAAAMg0A25ZU+E032AQEXoNcApCGFLX5paWlpaWl9dqq9AS6CKROfAAAAAElFTkSuQmCC\", \"http://localhost:5000\")","args":[]}] Failed to decode response from marionette
    Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
    System info: host: 'omg-6', ip: 'fe80:0:0:0:14cc:c9ee:ebd:ddc5%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.14.6', java.version: '13.0.2'
    Driver info: driver.version: unknown
      at Test.double image select works [as test] @ tests/functional/image-uploads-non-auth.js:219:10
      @ src/lib/Test.ts:268:51
      at runMicrotasks @ anonymous
```

To investigate later.

**Comment by miketaylr** [comment]
It seems like the workaround that was in place for a chromedriver bug crashed geckodriver. And all the tests pass with it removed, so...

File: tests/functional/image-uploads-non-auth.js [hunk 1]
```diff
@@ -213,14 +213,8 @@ registerSuite("Image Uploads (non-auth)", {
...
         .sleep(1000)
-        .execute(() => {
-          document.querySelector(".js-image-upload").click();
-        })
         .execute(POSTMESSAGE_TEST_SQUARE)
         .sleep(1000)
-        .execute(() => {
-          document.querySelector(".js-image-upload").click();
-        })
         .sleep(1000)
```
Comment (ksy36) [inline tests/functional/image-uploads-non-auth.js:218]: probably second `.sleep(1000)` could be removed?

Comment (miketaylr) [reply tests/functional/image-uploads-non-auth.js:218]: oops, thanks -- missed that!

**Pull request merged**